Presentation.Web/Controllers/API/V1/AdviceUserRelationController.cs

```diff
@@ -127,7 +127,7 @@ public HttpResponseMessage UpdateRange(int notificationId, RelatedEntityType rel
                 .Bind(ccs => MapRecipients(recipients, RecieverType.RECIEVER)
                     .Select(receivers => (ccs, receivers)))
                 .Bind(result => _registrationNotificationUserRelationsService.UpdateNotificationUserRelations(notificationId, result.ccs, result.receivers, relatedEntityType))
-                .Match(Ok, FromOperationError);
+                .Match(_=>Ok(), FromOperationError);
```

Comment:
@mrjsawdk This was the issue. It was returning the Advice object and that is an active object from the db... this is not an odata controller, so it all blew up